test/integration/targets/ec2_vol_facts/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+---
+# tasks file for test_ec2_vol_facts
+- name: Set up AWS connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: true
+
+- block:
+    - ec2_ami_facts:
+        <<: *aws_connection_info
+        filters:
+          architecture: x86_64
+          virtualization-type: hvm
+          root-device-type: ebs
+          name: "amzn-ami-hvm*"
+      register: amis
+
+    - name: Create test instance
+      ec2_instance:
+          name: "ansible_ec2_vol_facts_test"
```

Comment:
This (and any other resources) need to be using `{{ resource_prefix }}` on resource names, it's part of the requirements for the CI IAM policy.